SamplesDashboard/SamplesDashboard/ClientApp/src/views/Details.tsx

```diff
@@ -30,10 +48,10 @@ export default class Details extends React.Component<any, any> {
             { key: 'packageName', name: 'Library', fieldName: 'packageName', minWidth: 300, maxWidth: 400, isRowHeader: true, 
                 isResizable: true, isSorted: false, isSortedDescending: false, onColumnClick: this.onColumnClick
             },
-            { key: 'requirements', name: 'Used Version', fieldName: 'requirements', minWidth: 200, maxWidth: 300, 
+            { key: 'requirements', name: 'Used version', fieldName: 'requirements', minWidth: 200, maxWidth: 300, 
```

Comment:
>Used version [](start = 42, length = 12)

why do we want sentence casing instead of title casing here?

Reply:
@thewahome suggests that it improves user's readability and it is something they were also asked to implement on graph explorer for the same purpose.